localstack/cli/localstack.py

```diff
@@ -180,12 +180,17 @@ def cmd_stop():
 )
 @publish_invocation
 def cmd_logs(follow: bool):
+    from localstack.utils.bootstrap import LocalstackContainer
     from localstack.utils.docker_utils import DOCKER_CLIENT
 
     container_name = config.MAIN_CONTAINER_NAME
+    logfile = LocalstackContainer(container_name).logfile
 
     if not DOCKER_CLIENT.is_container_running(container_name):
         console.print("localstack container not running")
```

Comment:
We could improve the message here by indicating, when the logs are available, that these are the logs from the previous run